### ISSUE TITLE
LicenseProperties class not found in nuix-scripting-api-9.10.9.584.jar

### ISSUE THREAD

**Issue by VelpukondaRameshLio** [Nuix/Nuix-Java-Engine-Baseline#16]
Hi,

Latest nuix engine release 9.6.. version but licenseproperties class not found exception at compile time in jar nuix-scripting-api-9.10.9.584.jar

could you please check update ASAP.

Thank you

**Comment by JuicyDragon** [comment]
Hello @VelpukondaRameshLio I'm a little confused, are you using an engine release of version 9.10.9.584 or a 9.6.x release?  I've updated the code to support the latest release of 9.10.9.584 on the master branch of the repository.  I also see that the README still notes 9.6 so I will change that to the appropriate version.

If you need to develop against a Nuix 9.6 engine release, I have created a branch called [9.6.x](https://github.com/Nuix/Nuix-Java-Engine-Baseline/tree/9.6.x) which has a couple small changes to keep it working with a 9.6.x release, which primarily has to do with how Gradle resolves dependency JAR files (due to a naming change that occurred at some point).

Basically, if you wish to use the latest version, use the master branch, if you wish to use a 9.6 release, use the `9.6.x` branch.  Please let me know if this addresses your issue.  If not, please elaborate your problem a bit more for me so I know what to look at.  Thanks!
